src/test/java/com.endava.internship.collections/StudentMapTest.java

```diff
@@ -0,0 +1,235 @@
+package com.endava.internship.collections;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class StudentMapTest {
+
+    public static List<Student> students;
```

Comment:
no need to be public here, rather private. Also if it was me I would not use method BeforeAll, just used here the List.of() and make it a constant

Reply:
I agree, but I don't know why List.of() doesn't work. I used Stream.of(...).collect(Collectors.toList())

Reply:
as I see from the pom, your java is automaticaly configured as 1.8, but List.of() appeared in 1.9 or later